src/icons/general/IconSearch.js.flow

```diff
@@ -6,7 +6,7 @@ import { bdlGray40 } from '../../styles/variables';
 
 import type { Icon } from '../flowTypes';
 
-const IconSearch = ({ className = 'icon-search', color = bdlGray40, height = 14, title, width = 14 }: Icon) => (
+const IconSearch = ({ className, color = bdlGray40, height = 14, title, width = 14 }: Icon) => (
     <AccessibleSVG
         className={`icon-search ${className}`}
```

Comment:
Does removing the default value all together potentially add `undefined` to the className prop on line 11? I don't remember exactly what happens but I think using `className = ''` on line 9 might be safer?

Reply:
Yeah it looks like the `undefined` is causing the lint check to fail.